posts/2024-02-27-24.0.0.2.adoc

```diff
@@ -0,0 +1,332 @@
+---
+layout: post
+title: "Negative acknowledgment and more for MicroProfile Reactive Messaging in 24.0.0.2"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/dmuelle
+author_github: https://github.com/dmuelle
+seo-title: Negative acknowledgement and more for MicroProfile Reactive Messaging in 24.0.0.2 - OpenLiberty.io
+seo-description: The 24.0.0.2 release includes new capabilities for MicroProfile Reactive Messaging and MicroProfile Streams Operators, including negative acknowledgement, emitters, and backpressure support.
+blog_description: The 24.0.0.2 release includes new capabilities for MicroProfile Reactive Messaging and MicroProfile Streams Operators, including negative acknowledgement, emitters, and backpressure support.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Negative acknowledgment and more for MicroProfile Reactive Messaging in 24.0.0.2
+David Mueller <https://github.com/dmuelle>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+The 24.0.0.2 release includes new capabilities for MicroProfile Reactive Messaging and MicroProfile Streams Operators, including negative acknowledgment, emitters, and backpressure support. It also introduces a new Open Liberty guide to help you build true-to-production integration tests for your applications.
+
+
+In link:{url-about}[Open Liberty] 24.0.0.2:
+
+* <<mpreact, Negative acknowledgment and more for MicroProfile Reactive Messaging 3.0 and MicroProfile Streams Operators 3.0>>
+* <<tescont, New guide: "Building true-to-production integration tests with Testcontainers">>
+* <<CVEs, Security Vulnerability (CVE) Fixes>>
```

Comment:
Since there are no CVEs fixed in this release, should this section be taken out?